lib/sass/rails/cache_store.rb

```diff
@@ -1,31 +0,0 @@
-require 'sass'
-
-module Sass
-  module Rails
-    class CacheStore < ::Sass::CacheStores::Base
```

Comment:
Yes. The one that is defined here: https://github.com/rails/sprockets/blob/master/lib/sprockets/sass_cache_store.rb